rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/BaseWorkspaceService.java

```diff
@@ -108,11 +110,12 @@ public void didChangeWorkspaceFolders(DidChangeWorkspaceFoldersParams params) {
 
     @Override
     public CompletableFuture<Object> executeCommand(ExecuteCommandParams params) {
-        if (params.getCommand().startsWith(RASCAL_META_COMMAND)) {
+        if (params.getCommand().startsWith(RASCAL_META_COMMAND) || params.getCommand().startsWith(RASCAL_COMMAND)) {
```

Comment:
Yes we are, because otherwise the LSP would break. In fact we were matching the wrong name here (and als passing the wrong name) and things got jumbled up between the two LSP servers.